package.json

```diff
@@ -73,8 +77,10 @@
     "@commitlint/config-conventional": "^12.1.1",
     "@pancakeswap/eslint-config-pancake": "^1.1.0",
     "@types/jest": "^26.0.23",
+    "@types/react-datepicker": "^3.1.8",
```

Comment:
types for deps shouldn't be in the dev deps. I think if you try to build in prod mod it will fail